### PR TITLE
feat(indexer): Add prisma ui

### PR DESCRIPTION
**Pull request by roninjin10**
Add prisma studio ui
image
image
image

### CODE REVIEW

File: indexer/ui/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,109 @@
+generator client {
```
Comment (roninjin10) [inline indexer/ui/schema.prisma:1]: This file is generated

**Comment by roninjin10** [comment]
DX-62

**Comment by roninjin10** [comment]
https://linear.app/optimism/issue/DX-62/maintainability-add-ui-for-exploring-the-db

File: indexer/docker-compose.yml [hunk 2]
```diff
@@ -38,13 +38,27 @@ services:
       - INDEXER_REST_HOSTNAME=0.0.0.0
       - INDEXER_REST_PORT=8080
       - INDEXER_BEDROCK_L1_STANDARD_BRIDGE=0
-      - INDEXER_BEDROCK_L1_STANDARD_BRIDGE=0x636Af16bf2f682dD3109e60102b8E1A089FedAa8
-      - INDEXER_BEDROCK_OPTIMISM_PORTAL=0xB7040fd32359688346A3D1395a42114cf8E3b9b2
+      - INDEXER_BEDROCK_L1_STANDARD_BRIDGE=${INDEXER_BEDROCK_L1_STANDARD_BRIDGE:-0x636Af16bf2f682dD3109e60102b8E1A089FedAa8}}
```
Comment (nickbalestra) [inline indexer/docker-compose.yml:41]: should that trailing double `{` be there ?

**Review by nickbalestra — CHANGES_REQUESTED** [review]

File: indexer/docker-compose.yml (hunk 2 above)
Comment (roninjin10) [reply indexer/docker-compose.yml:41]:
```suggestion
      - INDEXER_BEDROCK_L1_STANDARD_BRIDGE=${INDEXER_BEDROCK_L1_STANDARD_BRIDGE:-0x636Af16bf2f682dD3109e60102b8E1A089FedAa8}
```
Nope definitely a copy paste bug

**Review by nickbalestra — APPROVED** [review]

**Pull request merged**